### DISCUSSION TITLE
Add the objectIdentifier to the Entity name

### DISCUSSION THREAD

**Discussion by CV-Private** [Bepacom-Raalte/bepacom-HA-Addons#44] [Ideas]
I propose to add the objectIdentifier to the HA Entity-ID, to make it easier to find. 

For example, the BACnet object:

objectIdentifier : analogValue,1688
objectName : parZoneRoomManualSetpoint-3

should have the HA Entity-ID _gtw_21_bacnet_783351856__**AV1688**__parzoneroommanualsetpoint_3_, so that I can find the Entity by searching for "_AV1688_" in HA.

With its current Entity-ID _gtw_21_bacnet_783351856_parzoneroommanualsetpoint_3_, I'm not able to identify the Entity of analogValue 1688 without looking it up in the object list in the WebUI of the Add-On.

**Comment by GravySeal** [comment]
I think it's a good idea.
Some people prefer the description, some prefer the object name, but a true neutral would probably be the object identifier.
The entity ID automatically gets generated, so there's not a whole lot we can do about that. 
With an object identifier name for each entity, the name would be something like "analogValue 1688", and the ID would be something like: number.gtw_21_bacnet_783351856_analogValue_1688

The 2 current options are object name and description, so this would be option number 3 when configuring the integration.

**Comment by Bepacom-Raalte** [comment]
This is done with v0.2.0 for the integration.
